cmd/runhcs/container.go

```diff
@@ -477,41 +477,78 @@ func createContainer(cfg *containerConfig) (_ *container, err error) {
 			annotationPreferredRootFSType  = "io.microsoft.virtualmachine.lcow.preferredrootfstype"
 		)
 
-		bothOpts := &uvm.Options{
-			ID:                   vmID(c.ID),
-			Owner:                cfg.Owner,
-			AllowOvercommit:      parseAnnotationsBool(cfg.Spec.Annotations, annotationAllowOverCommit),
-			EnableDeferredCommit: parseAnnotationsBool(cfg.Spec.Annotations, annotationEnableDeferredCommit),
-		}
+		var (
```

Comment:
@kevpar - This part is ugly. Your opts code would help here a lot.

Reply:
Does it get simpler if we pass a default value to parseAnnotation*()? I left another comment on that below.